Update tasklist.c to fix incompatible pointer type error:

| libwnck/tasklist.c: In function 'sequence_timeout_callback':
| libwnck/tasklist.c:4922:49: error: passing argument 2 of 'sn_startup_sequence_get_last_active_time'
     from incompatible pointer type [-Wincompatible-pointer-types]
|  4922 |                                                 &tv_sec, &tv_usec);
|       |                                                 ^~~~~~~
|       |                                                 |
|       |                                                 long int *

Upstream-Status: Submitted [https://gitlab.gnome.org/GNOME/libwnck/-/merge_requests/56]

Signed-off-by: Kai Kang <kai.kang@windriver.com>

---
 libwnck/tasklist.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libwnck/tasklist.c b/libwnck/tasklist.c
index b773247..f195074 100644
--- a/libwnck/tasklist.c
+++ b/libwnck/tasklist.c
@@ -4919,7 +4919,7 @@ sequence_timeout_callback (void *user_data)
       WnckTask *task = WNCK_TASK (tmp->data);
 
       sn_startup_sequence_get_last_active_time (task->startup_sequence,
-                                                &tv_sec, &tv_usec);
+                                                (time_t *)&tv_sec, &tv_usec);
 
       elapsed = (now - (tv_sec * G_USEC_PER_SEC + tv_usec)) / 1000.0;
 
